pwndbg/arch.py

```diff
@@ -18,6 +18,7 @@
 import pwndbg.typeinfo
 
 current = 'i386'
+mode	= {CS_MODE_ARM:"ARM",CS_MODE_THUMB:"THUMB",None:""}[None]
```

Comment:
Okay. 
Although, I did it it avoid the naming confusion due to so many modes(mode, CapstoneMode, MODES). Also changing the variable name to anything other than mode would make it loose its meaning.